blog/2023-11-01-feature-flags-sdks-architectures.md

```diff
@@ -0,0 +1,159 @@
+---
+slug: 'feature-flags-sdks-architectures'
+title: 'Feature Flags services Server-side SDKs different architectures approaches'
+date: 2023-11-01
+authors: [liran2000]
+description: "Feature Flags services Server-side SDKs different architectures approaches"
+tags: [client, sdk]
+draft: false
+---
+
+Server-side feature flag software development kits (SDKs) are common for letting applications like microservices interact with Feature flag services.
+Feature Flags SDKs have several used functionalities, mainly feature flag evaluation with or without context.
+Each Feature Flags service can publish SDKs in multiple programming languages.
+The Feature Flags services commonly expose APIs via endpoints like REST HTTP and/or gRPC.
+SDKs are an important layer, as network traffic loads affect both the application and the Feature Flags services.
+When the Feature Flags service is a Cloud SaaS service, some services support a relay proxy or a sidecar application, which can be deployed on an organization network.
```

Comment:
Can we change to read:
When the feature flag service is a Cloud SaaS service, often it supports a relay proxy or a sidecar application, which can be deployed on an organization network.